src/recipes/build.ts

```diff
@@ -0,0 +1,150 @@
+import { CycliToolbox, Mode, Platform, ProjectContext } from '../types'
+import { join } from 'path'
+
+const createBuildWorkflowForAndroid = async (
+  toolbox: CycliToolbox,
+  context: ProjectContext,
+  { mode, expo }: { mode: Mode; expo: boolean },
+  workflowProps: Record<string, string> = {}
+): Promise<string> => {
+  const gradleCommands =
+    mode === 'debug' ? 'assembleDebug' : 'assembleRelease assembleAndroidTest'
```

Comment:
Maybe `Mode` should be enum

Reply:
I'd prefer to stick to types as alternatives of strings to be consistent with types like `PackageManager` and `LockFile`

Reply:
You can do something like:

```
const SOMETHING = {
    ONE: 'one',
    TWO: 'two',
} as const

type T = typeof SOMETHING[keyof typeof SOMETHING]
```

Then I think you can use it both ways